playbooks/bibdata.yml

```diff
@@ -13,6 +13,8 @@
   roles:
     - role: roles/bibdata
     - role: roles/bibdata_sqs_poller
+      # https://github.com/pulibrary/princeton_ansible/issues/3041
+      when: runtime_env | default('staging') !="qa"
```

Comment:
I think this is just a spacing issue. Try committing this and see if it works:
```suggestion
      when: runtime_env | default('staging') != "qa"
```

Reply:
@acozine you are right! I'm sorry they are all skipped :) . Thank you!

Reply:
I also ran it against staging to confirm that sqs-poller tasks are not skipped on staging.